rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -304,10 +297,236 @@ namespace rocal{
         m.def("rocalRelease",&rocalRelease);
         // rocal_api_types.h
         py::class_<TimingInfo>(m, "TimingInfo")
-            .def_readwrite("load_time",&TimingInfo::load_time)
-            .def_readwrite("decode_time",&TimingInfo::decode_time)
-            .def_readwrite("process_time",&TimingInfo::process_time)
-            .def_readwrite("transfer_time",&TimingInfo::transfer_time);
+            .def_readwrite("load_time", &TimingInfo::load_time)
+            .def_readwrite("decode_time", &TimingInfo::decode_time)
+            .def_readwrite("process_time", &TimingInfo::process_time)
+            .def_readwrite("transfer_time", &TimingInfo::transfer_time);
+        py::class_<rocalTensor>(m, "rocalTensor")
+                .def(
+                "batch_height",
+                [](rocalTensor &output_tensor)
+                {
+                    return output_tensor.info().max_shape().at(1);
+                },
+                R"code(
+                Returns a tensor buffer's height.
+                )code"
+            )
+            .def(
+                "batch_width",
+                [](rocalTensor &output_tensor)
+                {
+                    return output_tensor.info().max_shape().at(0);
+                },
+                R"code(
+                Returns a tensor buffer's width.
+                )code"
+            )
+            .def(
+                "batch_size",
+                [](rocalTensor &output_tensor)
+                {
+                    return output_tensor.info().dims().at(0);
+                },
+                R"code(
+                Returns a tensor batch size.
+                )code"
+            )
+            .def(
+                "color_format",
+                [](rocalTensor &output_tensor)
+                {
+                    switch(output_tensor.info().color_format())
+                    {
+                        case RocalColorFormat::RGB24:
+                            return 3;
+                        case RocalColorFormat::BGR24:
+                            return 3;
+                        case RocalColorFormat::RGB_PLANAR:
+                            return 3;
+                        case RocalColorFormat::U8:
+                            return 1;
+                    }
+                },
+                R"code(
+                Returns a tensor batch size.
+                )code"
+            )
```

Comment:
We need to have an API which will return the dims

Reply:
Added